operator/model/src/main/java/io/apicurio/registry/operator/api/v1/ApicurioRegistry3Spec.java

```diff
@@ -1,17 +1,36 @@
 package io.apicurio.registry.operator.api.v1;
 
-import com.fasterxml.jackson.annotation.JsonInclude;
-import com.fasterxml.jackson.annotation.JsonPropertyOrder;
-import com.fasterxml.jackson.databind.JsonDeserializer;
+import com.fasterxml.jackson.annotation.*;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.databind.JsonDeserializer.None;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import io.fabric8.kubernetes.api.model.KubernetesResource;
-import lombok.Getter;
-import lombok.Setter;
+import lombok.*;
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
-@JsonPropertyOrder({ "configuration", "deployment" })
-@JsonDeserialize(using = JsonDeserializer.None.class)
+@JsonInclude(Include.NON_NULL)
+@JsonPropertyOrder({ "appHost", "uiHost" })
+@JsonDeserialize(using = None.class)
+@NoArgsConstructor
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@Builder
 @Getter
 @Setter
+@ToString
 public class ApicurioRegistry3Spec implements KubernetesResource {
+
+    /**
+     * Apicurio Registry backend base URL
+     */
+    @JsonProperty("appHost")
+    @JsonPropertyDescription("Apicurio Registry backend base URL")
```

Comment:
base ?

Reply:
Meaning that it's not URL directly to the API, so it's not directly usable in e.g. serdes.

Reply:
oh, ok ... not sure what would be a better name though